Systems.org

```diff
@@ -1000,12 +1000,13 @@ Once you have your Linux root partition set up, you can enable LUKS to encrypt t
 
 #+end_src
 
-Finally, make sure to mount your EFI partition to =/mnt/boot= so that the installer can install the bootloader.  The Guix installation instructions obscure this step slightly so it's easy to miss:
+Finally, make sure to create and mount your EFI partition to =/mnt/boot= so that the installer can install the bootloader.  The Guix installation instructions obscure this step slightly so it's easy to miss:
 
 #+begin_src sh
 
+  mkfs.vfat -n GUIX-BOOT /dev/<EFI device>
   mkdir -p /mnt/boot/efi
-  mount /dev/<EFI partition> /mnt/boot/efi
+  mount LABEL=GUIX-BOOT /mnt/boot/efi
```

Comment:
Creation of FAT32 EFI partition was missing.

Reply:
On my machine I don't have to create one because it already exists, though I can see why it would be good to mention that one may need to be created if it doesn't exist already.

Reply:
It already exists? I always to manually create all my partitions, anyways I usually work on a clean drive, (I manually delete any boot/EFI partitions because previous installations usually mess up with my bootloader entries).

If you buy a new drive or just clean it up completely you'd still need to partition everything.

Reply:
Yep, I usually don't have the luxury of a clean drive because I have to dual-boot with Windows.